node/node.go

```diff
@@ -259,10 +259,13 @@ func (n *NodeController) controlLoop(ctx context.Context) error {
 			return nil
 		case updated := <-n.chStatusUpdate:
 			var t *time.Timer
+			var resetDuration time.Duration
```

Comment:
Maybe we can set this before we start the loop?

Reply:
I've made this change in the new commit, please take a look.

Personally I think it was more obvious why the reset duration had to be changed when it was bundled with this
https://github.com/virtual-kubelet/virtual-kubelet/blob/ba940a9739fe0c661f9edfa3192ad84a412e23a3/node/node.go#L262-L266
logic.

Maybe selecting the active timer could be moved out of the loop as well?